### PR TITLE
More CI builds and SpotBugs checks

### PR DESCRIPTION
**Pull request by kondratyev-nv**

### CODE REVIEW

File: src/test/java/io/github/mfvanek/pg/utils/DatabasePopulator.java [hunk 1]
```diff
@@ -313,13 +313,13 @@ private void createTableWithoutPrimaryKey() {
...
                     "id bigint not null, " +
                     "name varchar(255) not null)", schemaName));
-            final String checkQuery = String.format("select exists (\n" +
-                    "   select 1 \n" +
-                    "   from pg_catalog.pg_class c\n" +
-                    "   join pg_catalog.pg_namespace n on n.oid = c.relnamespace\n" +
-                    "   where n.nspname = '%s'\n" +
-                    "   and c.relname = '%s'\n" +
-                    "   and c.relkind = 'r'\n" +
+            final String checkQuery = String.format("select exists (%n" +
```
Comment (mfvanek) [inline src/test/java/io/github/mfvanek/pg/utils/DatabasePopulator.java]: Something strange here. Please revert these changes

File: src/main/java/io/github/mfvanek/pg/model/IndexWithNulls.java [hunk 2]
```diff
@@ -36,6 +37,26 @@ public String toString() {
                 '}';
     }
 
+    @Override
+    public boolean equals(Object o) {
```
Comment (mfvanek) [inline src/main/java/io/github/mfvanek/pg/model/IndexWithNulls.java]: Please cover all new `equals` and `hashCode` methods with proper unit tests

File: src/test/java/io/github/mfvanek/pg/utils/DatabasePopulator.java (hunk 1 above)
Comment (kondratyev-nv) [reply src/test/java/io/github/mfvanek/pg/utils/DatabasePopulator.java]: This is SpotBugs bug VA_FORMAT_STRING_USES_NEWLINE.  "%n" always outputs the correct platform-specific line separator, it is portable across platforms whereas"\n" is not, see https://stackoverflow.com/a/1883364/4182275.

File: src/main/java/io/github/mfvanek/pg/model/IndexWithNulls.java (hunk 2 above)
Comment (kondratyev-nv) [reply src/main/java/io/github/mfvanek/pg/model/IndexWithNulls.java]: Added
